package/AUTHORS

```diff
@@ -42,6 +42,7 @@ Chronological list of authors
   - Lukas Stelzl
   - Jinju Lu
   - Joshua L. Phillips
+  - David van der Spoel
```

Comment:
There's a good chance that I set @dspoel as the author of commit 54ed4b38b20351fd3c51c93854d334fb8aa70d03 because these were patches taken directly from git://git.gromacs.org/libxdrfile.git (see commit message) and IIRC they were authored by him.